apps/web/src/app/[locale]/(dashboard)/[wsId]/documents/[documentId]/page.tsx

```
@@ -206,27 +206,27 @@ export default function DocumentDetailsPage({ params }: Props) {
+          docId={documentId}
+          document={document.content as JSONContent}
+          provider={provider}
+        />
+      </Room>
+
+      {document.id && (
+        <DocumentShareDialog
+          isOpen={isShareDialogOpen}
+          onClose={() => setIsShareDialogOpen(false)}
+          documentId={document.id}
+          isPublic={document.is_public!!}
```

Biome (1.9.4)

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)